catalog/transformer/catalog-transformer-xml-binding/src/main/resources/result.xsd

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsd:schema xmlns:mc="urn:catalog:metacard" xmlns="urn:catalog:result"
+        xmlns:xsd="http://www.w3.org/2001/XMLSchema" targetNamespace="urn:catalog:result"
+        elementFormDefault="qualified" attributeFormDefault="unqualified"
+        version="1.0.0">
+
+	<!-- bring in other schemas -->
+    <xsd:import schemaLocation="metacard.xsd" namespace="mc"/>
+
+    <xsd:element name="result">
+		<xsd:complexType>
+			<xsd:sequence>
+				<xsd:element name="relevance" type="xsd:double" minOccurs="0">
+					<xsd:annotation>
+						<xsd:documentation>A "relevance" score (typical range of 0-1, values over 1 are those that have been boosted) is used to specify a sort policy based on relevance.
+						</xsd:documentation>
+					</xsd:annotation>
+				</xsd:element>
+				<xsd:element name="distance" type="xsd:double" minOccurs="0">
+					<xsd:annotation>
+						<xsd:documentation>The "distance" is a representation of the distance in meters, null if not part of a search that was sorted by distance.
+						</xsd:documentation>
+					</xsd:annotation>
+				</xsd:element>
+				<xsd:element ref="mc:metacard">
```

Comment:
Shouldn't this be a minOccurs of 1 and maxOccurs of 1?  Metacard ID is a required attribute so there should always be at least one metacard attribute for a result.